SPECS/scossl/scossl.spec

```diff
@@ -0,0 +1,50 @@
+%define debug_package %{nil}
+Summary:        The SymCrypt engine for OpenSSL (SCOSSL) allows the use of OpenSSL with SymCrypt as the provider for core cryptographic operations
+Name:           scossl
+Version:        0.1
+Release:        1%{?dist}
+License:        MIT License
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System/Libraries
+URL:            https://github.com/microsoft/SymCrypt-OpenSSL
+Source0:        %{name}-%{version}.tar.gz
```

Comment:
Please use an URL for `Source0` pointing to the original source. This should match with the URL from the entry inside `cgmanifest.json` for this spec.